packages/mw-log-summary/src/index.js

```diff
@@ -0,0 +1,40 @@
+import assign from 'lodash/assign';
+import pick from 'lodash/pick';
+
+function requestResponseLogBody(req, res) {
+  return { req, res: { body: res.body } };
+}
+
+function isRequest(req) {
+  return req.headers.type === 'request' || req.headers.type === 'reactive';
+}
+
+export default function mwLogSummary(req, next, options, log) {
+  const prefix = `stellar ${req.headers.type}`;
+  let start = null;
+
+  if (!isRequest(req)) {
```

Comment:
Will it log the enqueue and dequeue of the responses (to requests)? I think it should skip it

Reply:
it doesnt log request/responses. it does this complexity because its a middleware built for both pubsub and request/response